sdk/go/common/workspace/project.json

```diff
@@ -317,4 +322,4 @@
             }
         }
     }
-}
\ No newline at end of file
+}
```

Comment:
my editor has opinions about newlines at the end of files. idk if they're the right ones.